5-aragon-governance-proposal-process.tex

```diff
@@ -113,10 +108,8 @@ \subsection{The Requirements for Proposals}
 
 \begin{enumerate}
 	\item \textbf{Public deliberation:} all proposals must be shared during the public deliberation phase in the Aragon Forum and linked in the Aragon Discord Server, unless these services are unavailable.
-	\item \textbf{Voting period:} the vote on \gls{AragonVoice} must be at least 7 days.
+	\item \textbf{Voting period:} the vote must be at least 7 days.
```

Comment:
Are there other characteristics / requirements, except the duration, that the vote should have?